core/src/main/java/io/questdb/std/ObjIntHashMap.java

```diff
@@ -67,20 +69,20 @@ public final void clear() {
         Arrays.fill(keys, noEntryValue);
     }
 
-    public void clear(int newCapacity) {
-        if (newCapacity <= capacity) {
+    public void resetCapacity() {
```

Comment:
This method always clears values but name to me sounds that it is not mandatory, only some capacity manipulation is done.
Perhaps `reset()` would be better a name?

Reply:
Good idea. Renamed in a9aaa7df2e4c36edd7a1ce9ccaea0e9813ff481c